region/lowernorfair/east/Amphitheatre.json

```diff
@@ -230,6 +230,51 @@
       "flashSuitChecked": true,
       "devNote": "FIXME: Add acid-filled version of the Crystal Flash, and requirements to unlock the door with Missiles and Power Bombs."
     },
+    {
+      "link": [1, 1],
+      "name": "Carry G-Mode Back Through",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        {"or": [
+          "SpaceJump",
+          "canConsecutiveWalljump"
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithGMode": {
+          "morphed": false
+        }
+      },
+      "bypassesDoorShell": true,
+      "flashSuitChecked": true,
+      "devNote": "This should be possible with a precise and very long canDoubleBombJump, or strats with lots of acid damage."
```

Comment:
You can barely make it up in time without a jumpIntoIBJ if you do the DoubleIBJ very well.  With a jump start just need to not drop the IBJ.